cpp/models/ode_secir/model.h

```diff
@@ -35,10 +35,17 @@ namespace osecir
 
 // Create template specializations for the age resolved
 // SECIHURD model
-
-class Model : public CompartmentalModel<InfectionState, Populations<AgeGroup, InfectionState>, Parameters>
+template <class I = InfectionState>
+using Flows =
+    FlowChart<Flow<I, I::Susceptible, I::Exposed>, Flow<I, I::Exposed, I::InfectedNoSymptoms>,
+              Flow<I, I::InfectedNoSymptoms, I::InfectedSymptoms>, Flow<I, I::InfectedNoSymptoms, I::Recovered>,
+              Flow<I, I::InfectedSymptoms, I::InfectedSevere>, Flow<I, I::InfectedSymptoms, I::Recovered>,
+              Flow<I, I::InfectedSevere, I::InfectedCritical>, Flow<I, I::InfectedSevere, I::Recovered>,
+              Flow<I, I::InfectedCritical, I::Dead>, Flow<I, I::InfectedCritical, I::Recovered>>;
```

Comment:
Is there a nice way to make it more readable? I would prefer not to use a macro, but otherwise I see no way too shorten this definition.

Reply:
no idea. just don't like our clang format settings (or automatic clang-format) :) 
maybe there is a small setting for template arguments that doesn't reformat everything?

Reply:
Setting `BinPackParameters : false` moves all template parameters into a single line, which definitely looks better. However, this also affects function definitions, e.g.
```
    void eval_right_hand_side(Eigen::Ref<const Eigen::VectorXd> pop, Eigen::Ref<const Eigen::VectorXd> y, double t,
                              Eigen::Ref<Eigen::VectorXd> dydt) const
```
becomes
```
    void eval_right_hand_side(Eigen::Ref<const Eigen::VectorXd> pop,
                              Eigen::Ref<const Eigen::VectorXd> y,
                              double t,
                              Eigen::Ref<Eigen::VectorXd> dydt) const
```
and I don't think we can seperate function/template params (see documentation [here](https://clang.llvm.org/docs/ClangFormatStyleOptions.html#binpackparameters) - notice the lack of template specific options). Do we want this?

Of course, we can still disable clang-format and arrange the Flows manually, but that is probably a bad idea.
```
// clang-format off
...
// clang-format on
```

Reply:
> Of course, we can still disable clang-format and arrange the Flows manually, but that is probably a bad idea.

Is it? Disabling clang-format should certainly be the exception, but in cases like this it might help